src/scripts/eos-analysis

```diff
@@ -397,6 +397,10 @@ Runs a list of subcommands.
         help = 'The base directory for the storage of data files. Can also be set via the EOS_BASE_DIRECTORY environment variable.',
         dest = 'base_directory', action = 'store', default = get_from_env('EOS_BASE_DIRECTORY', './')
     )
+    parser_run.add_argument('-d', '--dry-run',
+        help = 'Whether or not to perform a dry run of the analysis.',
```

Comment:
Can you give a more user-friendly definition? the notion of `dry-run` may be unknown by the users.

Reply:
Will do!